pkg/wallet/api.go

```diff
@@ -15,6 +15,7 @@ import (
 // Wallet interface
 type Wallet interface {
 	Crypto
```

Comment:
So eventually this should have a Crypto, but rather some key creator / management.

We will also need a Decrypter I guess?

Reply:
Yes, we should end up with key management interface and decrypter interface.

Reply:
It doesn't need to have the Decrypt interface either, baha's getting rid of that in his PR because the way we ended up implementing decryption didn't make sense with a function of that name.

You can keep it or remove it, it should be gone soon either way.